### ISSUE TITLE
./run.bat start not working on windows

### ISSUE THREAD

**Issue by eskayML** [Significant-Gravitas/Auto-GPT#2476]
Hello maintainers and contributors, So I've tried running the `./run.bat start` which was written in the documentation  on both powershell and cmd,

I also tried using both while running as an administrator, they apparently aren't running

![image](https://user-images.githubusercontent.com/71832907/232945278-a3002662-ae30-49a8-9c3d-da4620ef36df.png)
![image](https://user-images.githubusercontent.com/71832907/232945316-be7c9319-3eb7-491f-bef9-8a1c93d8b166.png)

**Comment by ParsingMAINFRAME** [comment]
+1

**Comment by nickjava** [comment]
I get the same error on mac

**Comment by Nilok7** [comment]
Just type run.bat
Ignore the `start`, it runs without it.

**Comment by Explorergt92** [comment]
I've had the same troubles testing it in the .vscontainer using `./run.sh start` in the README.md and in PS with `./run.bat start` shown there too.

In the .devcontainer I can get it to run with error with 'bash run.sh`

```
vscode ➜ /workspaces/Auto-GPT (master) $ bash run.sh
Traceback (most recent call last):
  File "/workspaces/Auto-GPT/scripts/check_requirements.py", line 32, in <module>
    main()
  File "/workspaces/Auto-GPT/scripts/check_requirements.py", line 8, in main
    with open(requirements_file, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt\r'
run.sh: line 10: syntax error: unexpected end of file
```

There's a PR #2418 for it I tested but doesn't fix the issue for me. 

In PS C:\ `.\run.bat` starts it up without issue for me.

Using `python -m autogpt` works on in both.

**Comment by LanceLake** [comment]
@echo off
:loop
python scripts/check_requirements.py requirements.txt
if errorlevel 1 (
    echo Installing missing packages...
    pip install -r requirements.txt
)
python -m autogpt --speak --allow-downloads
goto loop

This is mine if you want to include looping and starting over if it crashes.

**Comment by ParsingMAINFRAME** [comment]
@Significant-Gravitas: I'm sure I'm a nobody to you but this GitHub has changed my life. I can't thank you enough for what you created here. Kinda cool you're in this remotes thing with me. I'm also not an experienced programmer and I believe you are a significant reason I'm changing that. T hank's for the inspiration, it is an honor. I'll probably be embarrassed i couldn't come up with another way to message you later but this was the best idea I could come up with. Also I'd really liike to know if I posted my personal API on the Master repo at some point, or multiple points, I really didn't mean to. I may have posted my SSN for all i realized. I've never felt so proud when I got this github to work lmao.
